Crab.go

```diff
@@ -0,0 +1,16 @@
+package crab
```

Comment:
Remove this `package crab` line

Reply:
i left it in cause the java/kotlin one also had `package`